### PR TITLE
[home] support time-series db dependency iotdb deploy doc

### PR DESCRIPTION
**Pull request by tomsun28**

### CODE REVIEW

File: home/i18n/en/docusaurus-plugin-content-docs/current/start/iotdb-init.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      password: root
+      # V_O_12 || V_0_13
+      version: V_0_13
+      # if iotdb version >= 0.13 use default queryTimeoutInMs = -1; else use default queryTimeoutInMs = 0
+      query-timeout-in-ms: -1
+      # default '7776000000'（90days,unit:ms,-1:no-expire）
+      expire-time: '7776000000'
+```
+
+### 常见问题
```
Comment (Ceilzcx) [inline home/i18n/en/docusaurus-plugin-content-docs/current/start/iotdb-init.md:60]: 这边不是也应该改成英文：FAQ

Comment (tomsun28) [reply home/i18n/en/docusaurus-plugin-content-docs/current/start/iotdb-init.md:60]: 是的 给漏了😂

**Comment by Ceilzcx** [comment]
需不需要markdown说明一下，优先使用V0.13.*版本；使用0.12.*及以下必须修改application.yml配置文件；不支持0.14.*版本

**Comment by hertzbeat** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
Do you need markdown to explain, the V0.13.* version is preferred; the application.yml configuration file must be modified when using 0.12.* and below; the 0.14.* version is not supported

**Comment by tomsun28** [comment]
> 需不需要markdown说明一下，优先使用V0.13.*版本；使用0.12.*及以下必须修改application.yml配置文件；不支持0.14.*版本

👍这个需要的 那我们就推荐V0.13.*版本

**Comment by hertzbeat** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
> Do you need markdown to explain, the V0.13.* version is preferred; the application.yml configuration file must be modified when using 0.12.* and below; the 0.14.* version is not supported

👍If this is needed, then we recommend the V0.13.* version

**Review by Ceilzcx — APPROVED** [review]

**Pull request merged**